zabbixsender/zabbixsnd/zabbix_sender.go

```diff
@@ -45,30 +44,27 @@ func (p *Packet) DataLen() ([]byte, error) {
 	return dataLen, nil
 }
 
+// Sender sends data to zabbix
+// Read more: https://www.zabbix.com/documentation/3.4/manual/config/items/itemtypes/trapper
 type Sender struct {
-	Host string
-	Port int
+	addr *net.TCPAddr
 }
 
```

Comment:
Nice idea to add address directly to Sender struct. Later on you will not have to use fmt.Sprintf() function to create the address from Host and Port.